src/Sylius/Bundle/AddressingBundle/spec/Form/Type/AddressTypeSpec.php

```diff
@@ -45,48 +45,48 @@ function it_has_a_valid_name()
     function it_builds_form_with_proper_fields(FormBuilder $builder)
     {
         $builder->addEventSubscriber(Argument::type(EventSubscriberInterface::class))
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('firstName', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('lastName', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('phoneNumber', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('company', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
-            ->add('country', 'sylius_country_choice', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->add('country', 'sylius_country_code_choice', Argument::any())
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('street', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('city', 'text', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
+            ->willReturn($builder)
+        ;
 
         $builder
             ->add('postcode', 'text', Argument::any())
-            ->shouldBeCalled()
```

Comment:
`shouldBeCalled` should stay when using builders. It's the only way we can check it was really invoked.